en_ewt-ud-dev.conllu

```diff
@@ -18275,8 +18275,8 @@
 11	my	my	PRON	PRP$	Number=Sing|Person=1|Poss=Yes|PronType=Prs	12	nmod:poss	12:nmod:poss	_
 12	teacher	teacher	NOUN	NN	Number=Sing	15	nsubj	15:nsubj	_
 13-14	havent	_	_	_	_	_	_	_	_
-13	have	have	AUX	VBP	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	15	aux	15:aux	_
-14	nt	not	PART	RB	_	15	advmod	15:advmod	_
+13	have	have	AUX	VBP	Mood=Ind|Number=Sing|Person=3|Tense=Pres|Typo=Yes|VerbForm=Fin	15	aux	15:aux	CorrectForm=has
+14	nt	not	PART	RB	Typo=Yes	15	advmod	15:advmod	CorrectForm=not
```

Comment:
Because it's spelled together with "have" I'd say `CorrectForm=n't`

Reply:
OK